src/types.ts

```diff
@@ -369,8 +372,8 @@ export const UPDATE_ACTIONS = {
 export type UpdateAction = typeof UPDATE_ACTIONS[keyof typeof UPDATE_ACTIONS];
 
 export type BehaviorOptions = {
-  newElement: Element;
-  behaviorElement: Element;
+  newElement?: Element;
+  behaviorElement?: Element;
```

Comment:
it's not apparent to me why we need this change?

Reply:
`HvComponentOptions ` is the type which populates the onUpdate. It defines these two values as optional.
https://github.com/Instawork/hyperview/blob/4a570ab01b3652fdf7840974bb50b0a958c724e8/src/types.ts#L180
https://github.com/Instawork/hyperview/blob/4a570ab01b3652fdf7840974bb50b0a958c724e8/src/types.ts#L206

I can't pass an optional value from `HvComponentOptions` into the required value in here (`BehaviorOptions`). Further, I'm not sure those two values are even needed, but this was the quickest way to get past the error.

You can see here that those values don't appear to be read from the opts at all
https://github.com/Instawork/hyperview/blob/4a570ab01b3652fdf7840974bb50b0a958c724e8/src/services/navigation/index.ts#L71-L74